integration-tests/chopsticks/package.json

```diff
@@ -19,22 +19,21 @@
     "eslint-config-prettier": "^9.1.0",
     "eslint-config-standard-with-typescript": "^43.0.1",
     "eslint-plugin-import": "^2.25.2",
+    "eslint-plugin-jsx-a11y": "^6.8.0",
     "eslint-plugin-n": "^15.0.0 || ^16.0.0 ",
     "eslint-plugin-prettier": "^5.1.3",
     "eslint-plugin-promise": "^6.0.0",
     "prettier": "^3.2.5",
     "ts-node": "^10.9.2",
     "tsx": "^4.7.1",
     "typescript": "*",
-    "vitest": "^1.4.0",
-    "eslint-plugin-jsx-a11y": "^6.8.0"
+    "vitest": "^1.4.0"
   },
   "scripts": {
     "ts-check": "tsc --noEmit",
     "lint": "eslint src && prettier --check src",
     "lint:fix": "eslint --fix src && prettier --write src",
     "clean": "rm -rf ./db",
-    "test": "LOG_LEVEL=error vitest",
-    "test:CI": "vitest --bail 0  --no-file-parallelism"
+    "test": "LOG_LEVEL=error vitest"
```

Comment:
Definitely lower priority, but we should investigate how to make the tests run faster without them throw errors. Our new GH pipeline currently takes almost 1h, but I have no idea if this refactoring will make tests go faster or slower.